test/test_torch.py

```diff
@@ -2477,6 +2477,38 @@ def test_var_mean_all_dims(self):
                 self.assertEqual(var1, var2)
                 self.assertEqual(mean1, mean2)
 
+    def test_std_mean_some_dims(self):
+        sizes = (4, 6, 7, 5, 3)
+        dims = len(sizes)
+        for device in torch.testing.get_all_device_types():
+            x = torch.rand(sizes, device=device)
+            for rng in range(2, dims):
```

Comment:
what is `rng` supposed to mean?  It doesn't look like a random number generator (the standard definition of rng)?

Reply:
Renamed